data/jobs.ts

```diff
@@ -0,0 +1,58 @@
+export const jobs = [
+  {
+    title: "Backend Engineer",
+    company: "Pollytech",
+    desc: "Pollytech is looking for someone to build out a public API for their custom ecommerce plugin. The API is a completely greenfield project, so there's flexibility to use the tools you enjoy, but python is preferred.",
+    location: "Youngstown, OH",
+    salary: "Negotiable",
+    contact: "Joe@CodeYoungstown.com",
+    date: "2022-08-20",
+  },
+  {
+    title: "Student Developer",
+    company: "Youngstown.io",
+    desc: "Youngstown.io is looking for student software developers to connect with local companies  to work on web and mobile apps on a contract basis. Flexible + fully remote. ~15 hours a week during contract. When applying, please tell us about the most awesome project you've built so far.",
+    location: "Youngstown, OH",
+    salary: "Starting at $15 an hour",
+    contact: "Joe@JoeDuncko.com",
+    date: "2022-08-20",
+  },
+  {
+    title: "Multiple Positions and Internships",
+    company: "Ultiumcells",
+    desc: "Hello, I am Austin Bacho Senior IT specialist for infrastructure out at Ultiumcells. We're looking for 4-5 fulltime people with experience. If you're still a student and looking for part time please also reach out. Please send me resumes at austin.bacho1@gmail.com and send me a text with your information and I can get it expedited. We're looking for people willing to learn global manufacturing system and application based support. If you speak Korean this is also a plus. Please reach out directly via my phone as I don't check my email account often. Thank you",
```

Comment:
Let's change this description to:

UltiumCells is looking for 4-5 fulltime people with experience. If you're still a student and looking for part time please also reach out. We're looking for people willing to learn global manufacturing system and application based support. If you speak Korean this is also a plus.